lib/file-perms.c

```diff
@@ -1,6 +1,6 @@
 /*
  * Copyright (c) 2012 BalaBit IT Ltd, Budapest, Hungary
```

Comment:
@bazsi I didn't bother to change it at this location, because the specified script will automatically correct all occurrences, including this one. Actually I did bother and corrected it by hand, but I stashed all manual corrections and decided to express it in sed. If you support such an upgrade, I'll submit the result of that script in a different PR.